x-pack/solutions/observability/plugins/synthetics/common/constants/synthetics/rest_api.ts

```diff
@@ -42,6 +42,7 @@ export enum SYNTHETICS_API_URLS {
   JOURNEY_SCREENSHOT = `/internal/synthetics/journey/screenshot/{checkGroup}/{stepIndex}`,
   DELETE_PACKAGE_POLICY = `/internal/synthetics/monitor/policy/{packagePolicyId}`,
   FILTERS = '/internal/synthetics/monitor/filters',
+  TRIGGER_TASK_RUN = '/internal/synthetics/trigger_task_run/{taskType}',
```

Comment:
Is there an api that exists already to this that isn't synthetics specific?

Reply:
no, task manager team suggested us to do this.